### ISSUE TITLE
Admin management endpoints

### ISSUE THREAD

**Issue by gondzo** [topcoder-platform/tc-message-service#64]
We need to add several admin endpoints for managing posts/topics:

1. New endpoint for allowing change of ownership (createdBy) for any post or topic e.g. POST /v5/topics/admin/ownership/edit with post or topic id and new owner id in payload. Should validate that new owner has access to the entity
2. New endpoint for initializing post_user_stats table for any post. This happens when there was some error while creating initial record for a newly created post in post_user_stats. e.g. POST /v5/topics/admin/posts/{postId}/stats/create with userId and action being in payload
3. New endpoint to allow updating any field for given post e.g. /v5/topics/admin/posts/{postId}/edit. Any attribute omitted from the request body should not be updated
4. New endpoint to allow updating any field for given topic e.g. /v5/topics/admin/topics/{topicId}/edit.  Any attribute omitted from the request body should not be updated

All admin endpoints should have /v5/topics/admin/* as prefix

@vikasrohit do we need task 1 at all since it will be covered by the endpoint 3 (can update any field)

**Comment by vikasrohit** [comment]
Yes, @gondzo we still need 1. We want the 3 and 4 to be a last resort. We would always encourage to use dedicated endpoint for the use case so that we can better track and avoid mishaps.  We would continue to add on specific endpoints as new use cases come along.

**Comment by vikasrohit** [comment]
Moving to backlog
